src/boot/boot.janet

```diff
@@ -753,27 +753,27 @@
   r)
 
 (defn compare=
-  "Equivalent of '=' but using compare function instead of primitive comparator"
+  ``Equivalent of `=` but using compare function instead of primitive comparator.``
```

Comment:
```suggestion
  ``Equivalent of `=` but using polymorphic `compare` instead of primitive comparator.``
```

Reply:
Nice. Not only the backticks, but adding "polymorphic" helps increase my understanding connects it to what I see in `(doc compare)`.